docs/development.md

```diff
@@ -141,7 +141,7 @@ Here is a list of commonly overriden configuration parameters (the full list is
 ```bash
 # Cluster settings.
 export CLUSTER_NAME="capz-cluster"
```

Comment:
should this be 
```suggestion
export AZURE_CLUSTER_NAME="capz-cluster"
```
?

Reply:
not sure, thought about it, but wasn't sure if we should change it

Reply:
I do believe CLUSTER_NAME should likely be common to all providers.

Reply:
discussed offline, let's keep this one common